Infoshare.Basics/Infoshare.Basics.FirstCode/Program.cs

```diff
@@ -176,18 +176,80 @@ static void Main(string[] args)
             // "nadwaga" gdy BMI > 24.9 i BMI <= 29.9
             // "otyłość" gdy BMI > 29.9
 
+            decimal weight = 91.5m;
+            decimal height = 1.8m;
+            //decimal BMI = weight * decimal.Parse(Math.Pow(2, 2));
+            decimal BMI = weight / (height * height); // czy jest jakiś ładnieszy sposób na zrobienie potęgi? Nie czaję, dlaczego nie można zrobić tego jak w linijce wyżej.
+
+            if (BMI < 18.5M)
+            {
+                Console.WriteLine("Niedowaga");
+            }
+            else if (BMI >= 18.5M && BMI <= 24.9M)
+            {
+                Console.WriteLine("Waga prawidłowa");
+            }
+            else if (BMI > 24.9M && BMI <= 29.9M)
+            {
+                Console.WriteLine("Nadwaga");
+            }
+            else
+            {
+                Console.WriteLine("Otyłość");
+            }
+
             //DODATKOWE
             //1. Papier kamień nożyce. Na wejściu przyjmij dwie wartości typu char, dopuszczalne wartości to
             //p - papier
             //k - kamień
             //n - nożyce
             //Jako wynik podaj rezultat rozgrywki. Jeżeli wygrywa gracz pierwszy zwróć "w1", jeżeli drugi - "w2", a przypadku remisu zwróć "r"
 
+            //Console.Write("Wybierz: P, K lub N: ");
```

Comment:
trochę inaczej zrobiłeś to zadanie niż w opisie :) powinno być zwrócone kto wygrał, czy pierwszy czy drugi "gracz"